src/app/inventory/__snapshots__/d2-stores.test.ts.snap

```diff
@@ -1939,7 +1939,7 @@ exports[`process stores generates a correct  Armor CSV export 1`] = `
     "Seasonal Mod": [
       "vaultofglass",
     ],
-    "Source": "vaultofglass",
```

Comment:
These are regressions we should either fix directly before merging this or file an issue for.

Reply:
working on this now